### PR TITLE
Publish landscape theme to npm

### PR DESCRIPTION
**Pull request by tomap**

### CODE REVIEW

**Review by SukkaW — COMMENTED** [review]
I am not sure but there should be existed publish-to-npm GitHub action that we could use.

**Comment by tomap** [comment]
I've just tested the npm version: 
https://github.com/tomap/hexo-theme-minidyne-demo/commit/279401397f15d907a9f20ab7d450f89af3d17aa3
and it seems to work: https://5fb1168ba4a40d00076527a0--hexo-theme-minidyne-demo.netlify.app/

**Review by stevenjoezhang — DISMISSED** [review]

**Review by SukkaW — DISMISSED** [review]
LGTM!

**Comment by tomap** [comment]
I gave it a more thoughts: 
* the publish on each commit to master won't work, especially if we don't increment the version
* we'll need the npm token in the github secrets
* we should probably publish when a new release is created

What do you think?

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Publish
+
+on: [push, pull_request]
```
Comment (stevenjoezhang) [inline .github/workflows/publish.yml:3]: How about using `create` or `release` here?

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#create
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release

Comment (tomap) [reply .github/workflows/publish.yml:3]: I don't have much time available. By all means, please edit this pr to your liking

**Review by 0xflotus — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (stevenjoezhang) [reply .github/workflows/publish.yml:3]: I do not have permission to access the settings page of this repo or set npm token in the github secrets. I will temporarily remove the workflow file, and then manually publish the new version of the landscape theme on npm.

**Review by stevenjoezhang — APPROVED** [review]

**Pull request merged**